src/scss/yellow/components/form/_steps.scss

```diff
@@ -20,9 +20,32 @@
     flex: 1;
     font-size: $size-font-xxs;
     line-height: $size-line-height-xxs;
-    padding: $size-spacing-xxs;
+    padding: 17px;
     text-align: center;
 
+    &:nth-of-type(2) {
+      @media screen and (min-width: $media-small) {
+        flex: 2;
+      }
+    }
+
+    @media screen and (min-width: 360px) {
+      padding: $size-spacing-xxs;
+    }
+
+    @media screen and (min-width: $media-medium) {
+      flex: unset;
+      padding: 17px 20px;
+    }
+
+    @media screen and (min-width: $media-normal) {
+      padding: 17px 36px;
+    }
+
+    @media screen and (min-width: $media-large) {
+      padding: 17px 48px;
```

Comment:
Can we have some tokens here, right?

$size-spacing-l = 48px
$size-spacing-m = 36px

See [Spacings](https://github.com/getninjas/niten-vscode-snippets#spacing)

Reply:
Yes, thank you! In this case I missed out the lines.